json-schema-form.d.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Shorthand to lookup for keys with `x-jsf-*` preffix.
+ */
+export function pickXKey(node: Object, key: 'presentation' | 'error-message'): Object | undefined;
+
+type JSConfig = {
+  initialValues?: Record<string, unknown>;
+  strictInputType?: boolean;
+  customProperties?: {
+    description: Function | string;
+    [key: string]: unknown;
+  };
+  inputTypes?: {
+    errorMessage: Record<string, unknown>;
+  };
+};
+
+type Fields = Record<string, unknown>[]; // TODO: We don't know what type we have here, we need to investigate
```

Comment:
```suggestion
type Fields = Record<string, unknown>[]; // TODO: Type the field based on the given JSON Schema properties.
```

Overall, let's not say "we don't know" because we do... (it's documented in JSDocs). We just don't have the time to type it properly right now 😅

Reply:
In the JSDoc, I saw the fields property try to use a type called Fields but it pointed to any, that's why I wrote we needed to investigate more.

Changed